packages/components/src/internal/components/entities/SingleParentEntityPanel.tsx

```diff
@@ -138,26 +133,22 @@ class SingleParentEntity extends PureComponent<SingleParentEntityProps> {
                         />
                     )}
                 </div>
-                {lcTypeName && chosenType && (
+                {chosenType && (
                     <QuerySelect
-                        componentId={'parentEntityValue_' + lcTypeName} // important that this key off of the schemaQuery or it won't update when the SelectInput changes
+                        componentId={'parentEntityValue_' + chosenType.label} // important that this key off of the schemaQuery or it won't update when the SelectInput changes
                         containerClass="row"
-                        formsy={false}
-                        label={capitalizeFirstChar(parentDataType.nounSingular) + ' IDs'}
                         inputClass="col-sm-6"
-                        labelClass="col-sm-3 col-xs-12 entity-insert--parent-label"
-                        name={'parentEntityValue_' + lcTypeName}
-                        onQSChange={this.onChangeParentValue}
+                        label={capitalizeFirstChar(parentDataType.nounSingular) + ' IDs'}
+                        labelClass="col-sm-3 col-xs-12 entity-insert--parent-label entity-insert--parent-select"
+                        multiple
+                        name={'parentEntityValue_' + chosenType.label}
                         onInitValue={this.onInitValue}
-                        preLoad
-                        loadOnChange // set to true so we'll reload to eliminate the last selected value from the list.
```

Comment:
I haven't looked at the QuerySelect component changes just yet, but I assume this `loadOnChange` prop was removed because it was no longer needed with the updated component?

Reply:
Good question. It has been supplanted by `clearCacheOnChange` which is set to `true` in `SelectInput.defaultProps`.